xDrip/Scenes/Entries/EntriesList/EntriesListViewController.swift

```diff
@@ -59,9 +74,89 @@ class EntriesListViewController: UIViewController, EntriesListDisplayLogic {
         interactor?.doLoad(request: request)
     }
     
+    @IBAction private func onCancelButtonTap() {
+        let request = EntriesList.Cancel.Request()
+        interactor?.doCancel(request: request)
+    }
+    
+    @IBAction func onEditButtonTap() {
+        isEdit.toggle()
+    }
+    
+    
     // MARK: Display
     
     func displayLoad(viewModel: EntriesList.Load.ViewModel) {
+        cellData = viewModel.cellData
+        tableView.reloadData()
+    }
+    
+    private func setupUI() {
+        editBarButtonItem.title = "edit".localized
+        
+        view.addBlur()
```

Comment:
Why do you need blur?

Reply:
due to designs
https://app.zeplin.io/project/5e7364d9646b0401cd71d267/screen/5e736502653b9cb4987930f4